wix-bookings/guides/Terminology.md

```diff
@@ -1,58 +1,57 @@
+| Integration | Extended functionality that's implemented by a 3rd-party service provider and made available to the merchant using Wix SPIs. For example, you, as a pricing provider, can offer custom varied pricing options to a Wix site using the [Wix Bookings Pricing Integration REST SPI](https://dev.wix.com/api/rest/wix-bookings/pricing-integration-spi). |
+| Locked session | A session for a class that customers can't book. For example, because the session is full, has a waitlist, or an administrator needs to manually screen each customer before adding them to the session. |
+| Member | Someone who is a registered [member](https://dev.wix.com/docs/rest/crm/members-contacts/members/members/introduction) in the Wix site and has a login. |
+| Merchant | The business that offers services to customers. The merchant has a Wix site for their business, unless your app is working with a [Headless](https://dev.wix.com/docs/go-headless/getting-started/about-headless/about-wix-headless) project. |
+| Option (Pricing) | The pricing possibilities for a service, which can vary based on factors such as the staff member, customer age, appointment time, or type of equipment. Each option has a list of supported choices and may affect the service price or have a uniform price. |
+| `PENDING` (Booking status) | Indicates that the booking is waiting to be confirmed or declined. Bookings in `PENDING` status are displayed in the business calendar. Bookings are automatically set as `PENDING` when an [eCommerce order](https://dev.wix.com/docs/rest/business-solutions/e-commerce/orders/introduction) related to the booking has been created. |
+| Policy (Bookings) | Rules set by the merchant regarding booking and canceling a service. These may include how far in advance customers can book, the deadline for cancellations, and the maximum number of participants allowed per booking. |
+| Pricing provider | A 3rd-party app that implements custom logic to determine the price of a booking with the [Pricing Integration Service Plugin](https://dev.wix.com/docs/rest/business-solutions/bookings/pricing/pricing-integration-spi/introduction). |
+| Resource | A business asset. For example, a staff member, a room, or equipment that's needed to provide a service. Each resource has a schedule that defines its availability. |
+| Schedule | A collection of all sessions associated with a specific service or resource. It includes information necessary to calculate availability. For more details, see appointment schedule, class schedule, and course schedule. |
+| Service | A business offering, that can be a class, course, or appointment-based. For example, a fitness studio may offer 30-minute 1-on-1 personal training sessions by appointment, 1-hour yoga classes every Monday at 7:00 PM, and introductory HIIT courses consisting of 3 training sessions on specific dates. For more information, see appointment, class, and course. |
```

Comment:
Small thing - I noticed sometimes we say "for more details" and sometimes we say "for more information" - we could try to be consistent.
Also, can we delete the word "more?"